components/director/internal/open_resource_discovery/validation.go

```diff
@@ -1030,6 +1036,30 @@ func validateAPIPartOfConsumptionBundles(value interface{}, targetURLs json.RawM
 	return nil
 }
 
+func validateDefaultConsumptionBundle(value interface{}, partOfConsumptionBundles []*model.ConsumptionBundleReference) error {
+	defaultConsumptionBundle, ok := value.(*string)
+	if !ok {
+		return errors.New("error while casting to defaultConsumptionBundle")
+	}
+
+	if defaultConsumptionBundle == nil {
+		return nil
+	}
+
+	var isFound bool
+	for _, bundleRef := range partOfConsumptionBundles {
+		if *defaultConsumptionBundle == bundleRef.BundleOrdID {
+			isFound = true
+			break
+		}
+	}
+
+	if !isFound {
+		return errors.New("defaultConsumptionBundle MUST be an existing option in the corresponding partOfConsumptionBundles array")
```

Comment:
I'm having some UX flashbacks with that uppercase MUST, we should probably lowecase it, since it's just _"must"_ in the other error messages

Reply:
Makes sense, totally.